h/views/home.py

```diff
@@ -33,3 +33,18 @@ def index(context, request):
         )
 
     return context
+
+
+@view_config(route_name='index',
+             request_method='GET',
+             feature='homepage_redirects')
+def index_redirect(context, request):
+    settings = request.registry.settings
+    redirect = settings.get('h.homepage_redirect_url',
+                            'https://web.hypothes.is')
```

Comment:
I think it would make more sense to redirect to `/search` by default, otherwise I will always get redirected to another domain in local development which is confusing.